src/LogikUI/Simulation/WidthMissmatchException.cs

```diff
@@ -4,24 +4,24 @@
 
 namespace LogikUI.Simulation
 {
-    public class WidthMissmatchException : Exception
+    public class InvalidProjectDataException : Exception
```

Comment:
This is not a good name for this exception. The exception is literally a `WidthMissmatchException` and any other name is not correct. This change should be reverted.

Reply:
Visual Studio rename error.